controllers/datasciencecluster/datasciencecluster_controller.go

```diff
@@ -179,9 +179,6 @@ func CreateReconciliationPlan(instance *dsc.DataScienceCluster) *profiles.Reconc
 	plan := &profiles.ReconciliationPlan{Components: make(map[string]bool)}
 
 	profile := instance.Spec.Profile
-	if profile == "" {
-		profile = profiles.ProfileCore
-	}
```

Comment:
shouldn't this be 
```
if profile == "" {
		profile = profiles.ProfileNone
	}
```
if explicitly remove ".spec.profile" from yaml.

Reply:
By putting the `default` key in the OpenAPI schema, the API server will mutate custom resources at creation time with the value. I figured it would be unnecessary to default inside the controller as well as at the k8s API. I can add it back in, if you'd prefer to keep the code-path there instead of on the CRD.

Reply:
you are right. i have tested the new image, when explicit remove the line from yaml and click "save" it automatically pop up to the yaml